### ISSUE TITLE
Error : Processing post-creation actions... The system cannot find the file specified

### ISSUE THREAD

**Issue by mtnrbq** [SAFE-Stack/SAFE-template#36]
When executing in PowerShell on Windows 10, in the post-creation actions I receive the error : The system cannot find the file specified.
In the template.json, there is the following in the postActions section: 
```json
"condition": "(OS != \"windows\")",
```
The only reference I've found to checking OS versions is [dotnet/templating](https://github.com/dotnet/templating/wiki/%22Runnable-Project%22-Templates) which has:
```json
"condition": "(OperatingSystemKind == \"Linux\")"
```
Not sure if the condition is not firing, and allowing the action to try and run the chmod action

Happy to do a PR with some clarification if I've misunderstood what's going on here

**Comment by theimowski** [comment]
@rmunn ideas?

**Comment by theimowski** [comment]
#34

**Comment by rmunn** [comment]
I copied that almost verbatim from @mlorbetske's code in https://github.com/dotnet/templating/pull/1042, and didn't have a Windows system to test on so I assumed that the `OS != "Windows"` condition was correct. But looking at [this bit of the `dotnet-templating` code](https://github.com/dotnet/templating/blob/04744d0c1e90b4b582baf5216f2f4e5663f1187d/test/Microsoft.TemplateEngine.Orchestrator.RunnableProjects.UnitTests/TemplateConfigTests/PostActionTests.cs), it looks like the `OS` variable in that condition should instead be `OperatingSystemKind`. Try making that change and see if that fixes it on Windows. (I can't test the change myself).

Or perhaps @mlorbetske will know what the right syntax should be here.

**Comment by theimowski** [comment]
Neither `OperatingSystemKind` nor `OS` works for me.. 
Let's wait and see if @mlorbetske could help here, otherwise I'll need to revert this change

**Comment by rmunn** [comment]
That particular post-action specifically runs the Unix `chmod` command, so it's supposed to only run on Linux and OS X. So it would be really nice if the dotnet templating docs would actually *document* this `OperatingSystemKind` parameter here.

Also, maybe try `OperatingSystemKind == \"Linux\" || OperatingSystemKind == \"Mac\"`. See if that condition works.

**Comment by theimowski** [comment]
`OperatingSystemKind == \"Linux\" || OperatingSystemKind == \"Mac\"` doesn't trigger for Win, but it also doesn't trigger for Linux Subsystem on Win.

Looks like none of `OperatingSystemKind` or `OS` variables are present in any OS

**Comment by rmunn** [comment]
Sigh... if we can't find a way to make that post-action only trigger on Linux/OS X, you'll have to revert this change, and I'll have to track down the three other PRs I made to other templates and revert them as well. Annoying.

Maybe someone at Microsoft will respond to https://github.com/dotnet/templating/issues/1421 and tell us how to *actually* do what we're trying to do (trigger a post-action only on Linux or OS X), and then we'll be able to revert the reversion and actually have a runnable `build.sh`. Here's hoping.

**Comment by theimowski** [comment]
Reverting the change for now

**Comment by rmunn** [comment]
If the condition is changed to `"(OS != \"Windows_NT\")"` (see https://github.com/dotnet/templating/issues/1421#issuecomment-367404828) does that prevent it from running on a Windows machine? I don't have a Windows machine available at the moment to test that, though I'll have one again on Monday.

**Comment by theimowski** [comment]
Fixed by #37
